contracts/mapping-token/contracts/darwinia/DarwiniaMappingTokenFactory.sol

```diff
@@ -109,16 +120,51 @@ contract DarwiniaMappingTokenFactory is Initializable, Ownable {
         require(info.source != address(0), "token is not created by factory");
         IERC20(token).mint(recipient, amount);
     }
+    
+    // cross transfer to remote chain without waiting any confirm information,
+    // this must request the burn proof can be always verified by the remote chain corrently.
+    // so, here the user's token burned directly.
+    function burnAndRemoteUnlock(uint32 specVersion, uint64 weight, address token, bytes memory recipient, uint256 amount) external payable {
```

Comment:
Do we need to keep two versions of implementation, might be better to remove later?